src/scroll-behavior.js

```diff
@@ -0,0 +1,61 @@
+import { wrap } from 'lodash';
+import ScrollBehavior from 'scroll-behavior';
+import Router from 'next/router';
+import Link from 'next/link';
+import setupHistory from './history';
+import StateStorage from './StateStorage';
+
+/* istanbul ignore next*/
+if (typeof window !== 'undefined') {
+    setupHistory();
+}
+
+const stateStorage = new StateStorage();
+const scrollBehaviorSymbol = Symbol('scrollBehavior');
+
+const getScrollBehavior = (options = { disableNextLinkScroll: true }) => {
+    /* istanbul ignore next*/
+    if (Router[scrollBehaviorSymbol]) {
+        return;
```

Comment:
```suggestion
        return Router[scrollBehaviorSymbol];
```

Can you add a test for this?